### PR TITLE
soundwire: schedule pm_runtime transition when Slaves do not report as ATTACHED

### PR DESCRIPTION
**Pull request by plbossart**
Fixes https://github.com/thesofproject/linux/issues/1361

@RanderWang can you check if this works for you?

### CODE REVIEW

File: drivers/soundwire/intel.c [hunk 1]
```diff
@@ -31,6 +31,7 @@
 
 #define SDW_INTEL_MASTER_DISABLE_PM_RUNTIME BIT(0)
 #define SDW_INTEL_MASTER_DISABLE_CLOCK_STOP BIT(1)
+#define SDW_INTEL_MASTER_DISABLE_PM_RUNTIME_IDLE (2)
```
Comment (ranj063) [inline drivers/soundwire/intel.c]: @plbossart you missed the BIT()?

**Comment by plbossart** [comment]
>   #define SDW_INTEL_MASTER_DISABLE_PM_RUNTIME BIT(0)
>   #define SDW_INTEL_MASTER_DISABLE_CLOCK_STOP BIT(1)
> +#define SDW_INTEL_MASTER_DISABLE_PM_RUNTIME_IDLE (2)
> 
> @plbossart <https://github.com/plbossart> you missed the BIT()?

You and I have this in common that we can't copy/paste a macro properly :-)
Thanks for noticing this, fixing now.

**Comment by RanderWang** [comment]
@plbossart   I tested on cml laptop.  It works well now.

**Comment by plbossart** [comment]
Thanks @RanderWang, merging now

**Pull request merged**